### PR TITLE
Add test chart

### PR DESCRIPTION
**Pull request by aerlaut**
# Background

Testing adding a new environment variable to the chart. This will not bother staging activities.

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock  (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/hms-dbmi-cellenics/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/hms-dbmi-cellenics/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/hms-dbmi-cellenics/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by ivababukova** [comment]
@aerlaut what is this?

File: charts/test/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+        env:
+        - name: K8S_ENV
+          value: {{ .Values.kubernetes.env | quote }}
+
+        - name: SANDBOX_ID
+          value: {{ .Values.biomageCi.sandboxId | quote }}
+
+        - name: RDS_CLUSTER_ID
+          value: {{ .Values.biomageCi.rdsClusterId | quote }}
```
Comment (aerlaut) [inline charts/test/templates/deployment.yaml:64]: The difference between `nodejs/chart` and this test chart is this block. This block should add a new environment variable to the staged API pod using the value passed to `biomageCi.rdsClusterId`.

**Comment by aerlaut** [comment]
@ivababukova this is a PR to create a test chart that I can use to test if the changes I made to the [API PR](https://github.com/hms-dbmi-cellenics/api/pull/323) and [biomage-utils PR](https://github.com/hms-dbmi-cellenics/biomage-utils/pull/81) will create a new environment variable in the API pod. This environment variable will be used by the API to decide which rds instance to connect to (default or a new instace).

**Comment by ivababukova** [comment]
I might be missing something out, but I don't understand why we need to make an entire chart for that. Can you not test the environment variable with the existing charts?

**Review by ivababukova — APPROVED** [review]
Since this is needed only temporary, for a test and then it will be removed, you can merge it to do your test.

**Pull request merged**